agent/internal/client/client.go

```diff
@@ -296,84 +292,25 @@ func (c *communicatorImpl) createCedarGRPCConn(ctx context.Context) error {
 			return errors.Wrap(err, "getting cedar config")
 		}
 
-		// TODO (EVG-14557): Remove TLS dial option fallback once cedar
-		// gRPC is on API auth.
-		catcher := grip.NewBasicCatcher()
 		dialOpts := timber.DialCedarOptions{
 			BaseAddress: cc.BaseURL,
 			RPCPort:     cc.RPCPort,
 			Username:    cc.Username,
 			APIKey:      cc.APIKey,
 			Retries:     10,
 		}
-		if runtime.GOOS == "windows" {
-			cas, err := c.getAWSCACerts(ctx)
-			if err != nil {
-				return errors.Wrap(err, "getting AWS root CA certs for cedar gRPC client connections on Windows")
-			}
-			dialOpts.CACerts = [][]byte{cas}
-		}
 		c.cedarGRPCClient, err = timber.DialCedar(ctx, c.cedarHTTPClient, dialOpts)
 		if err != nil {
-			catcher.Wrap(err, "creating cedar grpc client connection with API auth.")
+			return errors.Wrap(err, "creating cedar grpc client connection with API auth.")
 		} else {
```

Comment:
Do we still need this else clause? It seems like it's unnecessary now that TLS is out.